source/_docs/configuration/templating.markdown

```diff
@@ -466,7 +466,7 @@ The same thing can also be expressed as a test:
 ### Floors
 
 - `floors()` returns the full list of floor IDs.
-- `floor_id(lookup_value)` returns the floor ID for a given device ID, entity ID, area ID, or area name. Can also be used as a filter.
+- `floor_id(lookup_value)` returns the floor ID for a given floor name or alias, device ID, entity ID, area ID, or area name or alias. Can also be used as a filter.
```

Comment:
```suggestion
- `floor_id(lookup_value)` returns the floor ID for a given floor name or alias, area name or alias, entity ID or device ID. Can also be used as a filter.
```

Reply:
Missed area ID. :)
But I got main idea, make it perfectly organized.

Reply:
That was on purpose. Does it work with area id?

Reply:
You're right it doesn't! :)